### ISSUE TITLE
I'm doing everything true but miners doesn't listing to website.

### ISSUE THREAD

**Issue by akargokay4** [UnamSanctam/UnamWebPanel#76]
I'm doing everything true. My website php version is 7.4 and site is working but miners doesnt listing on website. What should i do ? Thx :)

**Comment by UnamSanctam** [comment]
Which host are you using for it now?

**Comment by akargokay4** [comment]
000webhost and im using php 8.0

**Comment by UnamSanctam** [comment]
Yeah then it should work fine, is the URL correct in the checker (no hidden new line or anything)? Sometimes 000webhost can also screw up the file permissions for the database file.

**Comment by akargokay4** [comment]
But there is no problem and i cant find the issue ooff

**Comment by UnamSanctam** [comment]
Check that there are write permissions to the database file.

**Comment by akargokay4** [comment]
Where is the write permission which you said on the 000webhost website

**Comment by UnamSanctam** [comment]
In the file manager you can probably find the setting, also check `__UNAM_LIB/Logs` for any error files.

**Comment by akargokay4** [comment]
There is no error file. I will be crazy dudeee

**Comment by UnamSanctam** [comment]
Delete all the files and upload clean ones, then check the `/api/endpoint.php` page to make sure it says: "Error: Invalid data." or something like that and try again.

**Comment by akargokay4** [comment]
okey im trying

**Comment by akargokay4** [comment]
it still not working

**Comment by UnamSanctam** [comment]
And the /api/endpoint.php page returns the correct data ("Error: Invalid data.")?

**Comment by akargokay4** [comment]
Yes

**Comment by UnamSanctam** [comment]
Can you try using the Example Settings here at the top of the wiki https://github.com/UnamSanctam/SilentCryptoMiner/wiki but just adding your site to the "API Endpoint URL" to test it with that?

**Comment by akargokay4** [comment]
There is a file in public_html folder defaultly. Should i delete it then copy to files in it or dont delete ?

**Comment by UnamSanctam** [comment]
Sure, delete it though it shouldn't really matter.

**Comment by akargokay4** [comment]
it says wrong password omg what did i do wrong

**Comment by akargokay4** [comment]
could you help me on discord i can share my screen

**Comment by akargokay4** [comment]
im changing the password from config file but it says password wrong

**Comment by UnamSanctam** [comment]
Try replacing the whole line with the default `$config['password'] = 'UnamSanctam';` and try that, you most likely could've accidentally added some extra character somewhere where it isn't supposed to be.

**Comment by akargokay4** [comment]
everything is right but it still says wrong password i cant find the issue anyway

**Comment by UnamSanctam** [comment]
Replace the config.php with the default one and then try copy pasting `UnamSanctam` into the password field. Make sure that you don't have caps lock or anything on.

**Comment by akargokay4** [comment]
Everything is working as must like be. How can i increase refresh rate of my webpanel ? It is refreshing data so slow.

**Comment by UnamSanctam** [comment]
The miner contacts the web panel every 1 minute, if it was faster then you'd have issues.

**Comment by akargokay4** [comment]
okey thanks for the information have a good day

**Comment by UnamSanctam** [comment]
You too.
